goodtables.yml

```diff
@@ -0,0 +1,9 @@
+datapackages:
+  - DataPackages/Goal/1/datapackage.json
+  # - DataPackages/Goal/2/datapackage.json
+  # - etc
+
+settings:
+  row_limit: 100000
```

Comment:
https://goodtables.io/github/roll/SDG/jobs/2

We have a validation warning. So increasing the row limit using the `settings` section.